src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/15.diff.sql

```diff
@@ -0,0 +1,98 @@
+﻿/*************************************************************
+    Background job checkpoint feature
+**************************************************************/
+
+/*************************************************************
+    Background job checkpoint table
+**************************************************************/
+
+IF NOT EXISTS (SELECT 1 FROM sys.tables WHERE name = 'BackgroundJobCheckpoint')
+BEGIN
+     CREATE TABLE dbo.BackgroundJobCheckpoint
+     (
+         Id bigint IDENTITY(1,1) NOT NULL,
+         UpdatedOn datetime2(7) NOT NULL DEFAULT sysutcdatetime(),
+         CheckpointId varchar(64) NOT NULL,
+         Partition varchar(64) NOT NULL,
+         LastProcessedDateTime datetimeoffset(7),
+         LastProcessedIdentifier varchar(64),
```

Comment:
Is there are document which describes how this table would be used?

Reply:
Created an internal spec